utils/format.sh

```diff
@@ -36,6 +36,10 @@ fix_format() {
     -o -name \*.cpp -print0 \
     -o -name \*.cl -print0 \
   | xargs -0 -P8 -n1 $CLANG_COMMAND -i;
+
+  find torch_glow/tests utils \
+    -name \*.py -print0 \
+  | xargs -0 autopep8 -j 8 -i -a;
```

Comment:
for `-j`, let's just do -1 or something right? 
> "number of parallel jobs; match CPU count if value is less than 1"

the autopep8 readme uses two `-a` (extra aggressive), should we do the same?